### PR TITLE
Typescript setup

### PR DESCRIPTION
**Pull request by rebzden**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "declaration": true,
+    "noImplicitAny": false,
+    "removeComments": true,
+    "noLib": false,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "target": "es6",
+    "sourceMap": true,
+    "lib": ["es6"]
+  },
```
Comment (ulfryk) [inline tsconfig.json:14]: where is `strict: true` ?

File: packages/core/src/core.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const x = true;
```
Comment (ulfryk) [inline packages/core/src/core.ts]: new line at the end of file :P

Comment (rebzden) [reply packages/core/src/core.ts]: fixed

File: tsconfig.json (hunk 1 above)
Comment (rebzden) [reply tsconfig.json:14]: fixed

**Review by ulfryk — APPROVED** [review]

**Pull request merged**